### PR TITLE
Release pipeline fix

### PR DESCRIPTION
**Pull request by ealsur**
# Pull Request Template

## Description

`BlobVersion` needs to be taken out in order to have the expected default value.

### CODE REVIEW

File: azure-pipelines-official.yml [hunk 1]
```diff
@@ -6,7 +6,6 @@ variables:
   ReleaseArguments: ' --filter "TestCategory!=Quarantine" --verbosity normal ' 
   VmImage: windows-latest # https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops 
   BuildConfiguration: Release
-  BlobVersion: ''
```
Comment (j82w) [inline azure-pipelines-official.yml]: Why do we need to override this?

Comment (ealsur) [reply azure-pipelines-official.yml]: That's the thing. According to the docs, if you define a variable in the YML, you can override it at the Queue level (when you queue the build), BUT, from practice, I found out that it's not the case, quite the opposite. If this variable is defined here with `''` as value, it cannot be overriden. So we need to remove it from here in order to be able to set it at Queue time.

**Pull request merged**